OutSystems/outsystems-datagrid-reactive
# RGRIDT-688: Implementing is mandatory validation

This PR implements the mandatory validation on columns

# What was happening

There was no is mandatory validation in columns.

# What was done

Created new input parameter inside Column containing the following structure (isMandatory + ErrorMessage).

IsMandatory has the following description: Boolean that tells if this column will be mandatory or not.If you wish to have another type of validation, you can use the SetValidationStatus client action.

ErrorMessage has the following description: Message that will be displayed if the cell is not filled with any value

# Test Steps

1. Open Sample

2. Delete content from a Product Name cell

Expected; Invalid mark should show with following text: “This cell is mandatory, please fill it.”




### Screenshots
(prefer animated gif)


### Checklist
* [x ] tested locally
